app/models/nic/interface.rb

```diff
@@ -74,9 +74,11 @@ def identifier_change
       old_value = self.identifier_was || ''
```

Comment:
@sean797 the problem is that interface is considered physical if identifier is not set. I tend to agree that until it's set, we can't reliably say what kind of interface it is. For interfaces imported from puppet we always have the identifier from facts so this should be safe.

But based on what you say, the problem is that we consider all interfaces with dots in name as virtual which is not true on s390x. So I wonder if we shouldn't rather consider `enccw.*` as physical interfaces. I'm not sure if all s390x identifiers start with this prefix though.